### PR TITLE
Develop Base Network

### PR DESCRIPTION
**Pull request by i-am-sijia**

### CODE REVIEW

**Comment by DavidOry** [comment]
Can we remove the `demo_pipeline` folder?

File: notebooks/pipeline/step4_conflate_with_third_party [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#docker build -t shst .
```
Comment (DavidOry) [inline notebooks/pipeline/step4_conflate_with_third_party:1]: @i-am-sijia: why not make this a shell script?

**Comment by DavidOry** [comment]
Are the methods in `methods.py` common enough that we should put them in (or aspire to put them in) `Lasso`?

**Comment by DavidOry** [comment]
@i-am-sijia 
What do all the `pems_lanes` fields mean? What is FF, FR, HV, ML, and OR?

**Comment by i-am-sijia** [comment]
The links can have different types of PeMS matches, like HOV and Mainline.  `pems_lanes_XX` is a dcast of the following:

[PeMS conflation result database](https://github.com/BayAreaMetro/travel-model-two-networks/blob/develop/data/interim/mtc/pems_conflation_result.geojson):
```
station, type, lanes, shstRefId
400000, ML, 4, 547yewif958awsfy
400000, HV, 1, 547yewif958awsfy
```

to links database:

```
shstRefId, pems_lanes_ML, pems_lanes_HV
547yewif958awsfy, 4, 1
```

PeMS type [dictionary](https://github.com/BayAreaMetro/pems-typical-weekday#data-dictionary):
> Lane type. Possible values: CD = Coll/Dist. CH = Conventional Highway. FF = Freeway-Freeway Connector. FR = Off Ramp. HV = HOV. ML = Mainline. OR = On ramp.

**Comment by i-am-sijia** [comment]
@DavidOry Shall we remove the 2016 GTFS when 2015 exists? e.g. [petalumatransit-petaluma-ca-us__11_12_15](https://github.com/BayAreaMetro/travel-model-two-networks/tree/master/data/external/gtfs/2015/petalumatransit-petaluma-ca-us__11_12_15) and [Petaluma_2016_5_22](https://github.com/BayAreaMetro/travel-model-two-networks/tree/master/data/external/gtfs/2015/Petaluma_2016_5_22), [westcat-ca-us_9_17_2015](https://github.com/BayAreaMetro/travel-model-two-networks/tree/master/data/external/gtfs/2015/westcat-ca-us_9_17_2015) and [WestCAT_2016_5_26](https://github.com/BayAreaMetro/travel-model-two-networks/tree/master/data/external/gtfs/2015/WestCAT_2016_5_26).

**Comment by DavidOry** [comment]
> @DavidOry Shall we remove the 2016 GTFS when 2015 exists? e.g. [petalumatransit-petaluma-ca-us__11_12_15](https://github.com/BayAreaMetro/travel-model-two-networks/tree/master/data/external/gtfs/2015/petalumatransit-petaluma-ca-us__11_12_15) and [Petaluma_2016_5_22](https://github.com/BayAreaMetro/travel-model-two-networks/tree/master/data/external/gtfs/2015/Petaluma_2016_5_22), [westcat-ca-us_9_17_2015](https://github.com/BayAreaMetro/travel-model-two-networks/tree/master/data/external/gtfs/2015/westcat-ca-us_9_17_2015) and [WestCAT_2016_5_26](https://github.com/BayAreaMetro/travel-model-two-networks/tree/master/data/external/gtfs/2015/WestCAT_2016_5_26).

Provided the 2015 data has shapes, please use it. If not, use 2016.

**Comment by DavidOry** [comment]
@i-am-sijia 
I'd like to bring this in `master` and then start another `develop` branch now that we have the shapes and IDs fairly well-established. Can you please check my last update from `master` to make sure I selected the right files? I'll then merge.

**Comment by i-am-sijia** [comment]
@DavidOry Done.

**Pull request merged**